.vib/osclass/goss/vars.yaml

```diff
@@ -0,0 +1,20 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/osclass
+      - /opt/bitnami/osclass
+      - /opt/bitnami/osclass/oc-content
+files:
+  - paths:
+      - /opt/bitnami/osclass/config-sample.php
```

Comment:
I would not check this file here as we have not performed any relevant operation to it. This is something that should come for granted in compilation in my opinion.

Reply:
Well, I think that we shouldn't take anything for granted. But it's true that chart tests won't work if this file does not exist, so I'll remove it.